app/static/admin/js/pages/analytics/graph/analytics-graph.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+define(["../analytics-util", "../constants"], function(util, c) {
+
+  function Graph(elementId, keyName, valueName, startDate, endDate, color) {
+    this._elementID = elementId;
```

Comment:
Is there a reason why these variables are underscored? I think it's fine without `_`. It just makes it more confusing imo.

Reply:
I guess its private. ok sure.